src/ralph/models/edx/base.py

```diff
@@ -15,6 +15,22 @@ class Config:  # pylint: disable=missing-class-docstring
         extra = "forbid"
 
 
+class ContextModuleField(BaseModelWithConfig):
+    """Represents the context `module` field.
+
+    Attributes:
+        usage_key (str): Consists of a block ID of the current component.
+        display_name (str): Consists of a short description or title of the component.
+    """
+
+    usage_key: constr(regex=r"^block-v1:.+\+.+\+.+type@.+@[a-f0-9]{32}$")  # noqa:F722
+    display_name: str
+    original_usage_key: constr(
+        regex=r"^block-v1:.+\+.+\+.+type@problem\+block@[a-f0-9]{32}$"  # noqa:F722
+    )
+    original_usage_version: str
```

Comment:
```suggestion
    original_usage_key: Optional[constr(
        regex=r"^block-v1:.+\+.+\+.+type@problem\+block@[a-f0-9]{32}$"  # noqa:F722
    )]
    original_usage_version: Optional[str]
```
The `original_usage_key` and `original_usage_version` fields are not always present